tools/cd_scripts/package_gcsfuse.sh

```diff
@@ -80,20 +84,21 @@ sudo apt-get install git -y
 # It is require for multi-arch support
 sudo apt-get install qemu-user-static binfmt-support
 
-git clone https://github.com/GoogleCloudPlatform/gcsfuse.git
+# TODO (raj-prince): to remove this branch once read-cache-release branched is merged into the master branch.
+git clone -b "beta_louhi_cd_script_final_change" https://github.com/GoogleCloudPlatform/gcsfuse.git
```

Comment:
Reminder: have to change "read_cache_release" branch.

Reply:
This should be master for non beta release?

Reply:
Yes.